.github/workflows/check-pr-title.yaml

```
@@ -0,0 +1,14 @@
+name: pr title check
+on:
+  pull_request:
+    types: [opened, edited, ready_for_review, reopened]
+jobs:
+  lint-pr-title:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false && github.event.pull_request.state == 'open'
+    permissions:
+      pull-requests: read
+    steps:
+      - uses: Slashgear/action-check-pr-title@v4.0.0
+        with:
+          regexp: '\[[A-Z]+\] .+'
```

Si on se concentre sur la partie Tag, est-ce qu'on ne devrait pas plutôt lister les types aussi de tags ? 
```suggestion
          regexp: '\[(TECH|FEATURE|BUGFIX|DOC)\] .+\.' 
```
Qu'en pensez-vous ?

---

La liste des tags est dispo dans le [contributing](https://github.com/1024pix/pix/blob/dev/CONTRIBUTING.md#tag).

Au choix, soit on contraint, soit on laisse libre avec le `[A-Z]+` pour inclure `DOC` et tous les autres mais qui ne servent pas pour générer le changelog (donc tombent dans la section "Autres").

Sur Pix UI certains utilisent `[BREAKING CHANGE]`, je ne pense pas que ce soit la bonne approche, mais c'est un constat qu'on a d'autres tags utilisés au global sur tous les projets.

IMO autant laisser libre sachant que ça n'a pas d'impact.

---

Le changelog indique qu'il est possible d'utiliser d'autres tags, donc je suis plutôt contre même si je ressens pas le besoin d'en avoir d'autres
> Il est possible d'utiliser d'autres tags mais le CHANGELOG les regroupera comme des modifications "Autres".

---

Actuellement, dans Autres, on peut voir :
DOC, CLEANUP, TASK, PERFORMANCE, A11Y, DOCUMENTATON (en plus de DOC), DX, BUG (au lieu de BUGFIX), ADMIN (donc l'app), API, etc.

Pour une première PR, permettre tout me parait OK, mais la fonctionnalité permettrait d'éviter d'avoir des doublons différents, ou alors des info HS (API, etc..).
Ca demande peut être par contre de modifier le contributing pour lister les quelques tags manquants qu'on souhaiterait ajouter.

Certains utilisent deux tags `[BUGFIX] [API]`: la regex ne l'empêche pas.

---

M'étant plusieurs fois posé la question pour de l'accessibilité et du Pix UI, je serais pour cadrer les tags.

---

Voir #5451 pour une discussion sur ce sujet en particulier :)